roles/deploy/hooks/finalize-after.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: WordPress Installed?
-  command: wp core is-installed {{ project.multisite.enabled | default(false) | ternary('--network', '') }}
+  shell: MULTISITE=false SUBDOMAIN_INSTALL=false wp core is-installed
```

Comment:
is `shell` needed for ENV vars?

Ansible does support `environment`: http://docs.ansible.com/ansible/playbooks_environment.html